### ISSUE TITLE
Compatibility with v1.2

### ISSUE THREAD

**Issue by orware** [oliviermaridat/kanboard-milestone-plugin#14]
Good evening,

I was just noticing after updating to v1.2 (and actually even with the previous 2 releases) it seems like the Milestone area isn't showing up for me anymore and I was wanting to find out if there were any issues using the plugin with the latest versions of Kanboard?

**Comment by oliviermaridat** [comment]
Hi!
Sorry, I did not follow the last versions of Kanboard, there is probably a breaking change. It is usually something very small to change.
I will take time to check this in January. If you prefer to dig now, please feel free to send a PR.

**Comment by orware** [comment]
Thank you for the reply! I'll try and take a look at it myself as well in the meantime but definitely appreciate you being able to take a look at it if I am not able to fix it on my own. We were looking to use it and I remembered having it installed and then noticed it wasn't really showing up in the interface anymore so I figured an incompatibility had come up since I first installed it.

**Comment by oliviermaridat** [comment]
I have tested this plugin today with the last version of Kanboard ([33 commits after 1.2.0](https://github.com/kanboard/kanboard/commit/da3d99324355ea1bfe2ece28751f0dec2f7bb441)) and it worked like a charm (see below). Can you try again please?

![image](https://user-images.githubusercontent.com/953989/36480092-54180e42-170b-11e8-995c-f8df6a6fa756.png)

**Comment by orware** [comment]
Thanks for the update @oliviermaridat.

That's interesting that it works with the current commit...I'll likely wait until the 1.2.1 release (or whatever the next one happens to be) to see how it goes for us but if that fixes things automatically then I guess that's a good thing :-).

I'll see if I can bring down a local copy and try it as well to see if I can replicate what you did.

**Comment by orware** [comment]
Hi @oliviermaridat,

I tested it out with the new 1.2.1 Kanboard Release but I'm still not seeing a Milestone section in my project. It's a bit weird why it doesn't show up (and I have the code present in the plugin folder and everything...it just kind of seems like it's either not being included when the page loads for some reason, or it's running into an error specific to my environment and that's preventing it from loading?).

**Comment by orware** [comment]
@oliviermaridat,

I apologize, but I may have simply been expecting there to be a Add Milestone option somewhere more obvious and not been following directions correctly. I looked at the Add Internal Link area and I was able to create the Milestone association between tasks today (when I initially created this issue it had been sometime since I had tried out the Milestone plugin initially so I think I simply forgot how the associations were actually added).

Sorry for misreporting that there was an issue.
